### ISSUE TITLE
Still not seeing power display (NVML: can't get power for GPU#0, not supported)

### ISSUE THREAD

**Issue by 186cx342ds3422** [trexminer/T-Rex#115]
This issue has been on going for a while now even with older driver versions. Warning shows on both Cuda 10 and Cuda 11.1 versions. Everyting else seems fine.

OS: Windows 10 Pro 20h2
NVIDIA Driver Version: 460.79
Graphic Card: EVGA RTX 2080 Super FTW3
OC Tool: EVGA Percision X1

![GPUZ_Info](https://user-images.githubusercontent.com/46388221/102141326-a4d87900-3e2e-11eb-8fb5-f28501229ec2.jpg)
![T-rex_Miner_Issue](https://user-images.githubusercontent.com/46388221/102141438-d2bdbd80-3e2e-11eb-8821-ed4422a21670.jpg)

**Comment by KielD-01** [comment]
@186cx342ds3422 I have got this solved by installing fresh GPU drivers with a clean installation option

**Comment by 186cx342ds3422** [comment]
Issue is resolved but for a slightly different reason...

I still hade the old Cuda DLL files in the T-Rex folder. Once I removed them and restarted the miner I'm able to get power stats like normal again.

Remove Below Files (or Rename each to .OLD)
nvml.dll
nvrtc64_100_0.dll
nvrtc64_110_0.dll
nvrtc64_111_0.dll
nvrtc-builtins64_100.dll
nvrtc-builtins64_110.dll
nvrtc-builtins64_111.dll

**Comment by BitBora** [comment]
I fixed it by installing driver 471.68 Studio Driver

**Comment by theluke** [comment]
Same problem here with:

```
GP107 [GeForce GTX 1050 Ti]
```
Ubuntu 21, Driver version
```
NVIDIA-SMI 470.86       Driver Version: 470.86       CUDA Version: 11.4 
```
I cannot seem to find a driver newer than `470` for Ubuntu
